client/src/components/web-extension/ImportWallet/ImportWallet.jsx

```diff
@@ -1,66 +1,29 @@
-import React, { useCallback } from 'react';
-import PropTypes from 'prop-types';
-import { Button } from 'react-bootstrap';
+import React, { useEffect } from 'react';
 import { useSelector } from 'react-redux';
-import { Formik, Form } from 'formik';
-import { selectCreateWalletTotalKeywords } from "@cd/selectors/createWallet";
-import FieldKeyphrase from "./FieldKeyphrase";
+import { useOutletContext } from 'react-router-dom';
+import { selectCreateWalletCurrentStep } from '@cd/selectors/createWallet';
+import CreatePasswordPage from '@cd/web-extension/Common/CreatePasswordPage';
+import ImportWalletPage from './ImportWalletPage';
 import './ImportWallet.scss';
 
-
 const ImportWallet = () => {
-	const TOTAL_KEYWORDS = useSelector(selectCreateWalletTotalKeywords);
-
-	const onValidate = useCallback(values => {
-		console.log(`🚀 ~ onValidate ~ values`, values)
-		const { keyphrase } = values;
-		const errors = {};
-
-    /**
-     * Make sure keyphrase has 12 words
-     */
-
-
-    /**
-     * Make sure keyphrase is valid with `KeyFactory.getInstance().validate(keyphrase)`
-     */
-
-		return errors;
-	}, []);
-
+	const [, setHeader] = useOutletContext();
+	const currentStep = useSelector(selectCreateWalletCurrentStep);
+	useEffect(() => {
+		if (currentStep === 0) {
```

Comment:
Consider converting this to switch case to avoid "if" many times :D

Reply:
Good one,

That's why in Creating wallet I did create a util for generating proper heading based on current step.  🤣